pkgs/applications/misc/syncthingtray/default.nix

```diff
@@ -0,0 +1,50 @@
+{ mkDerivation, stdenv, lib, fetchFromGitHub
+, qtbase, extra-cmake-modules, cpp-utilities, qtutilities
+, cmake, kio, plasma-framework, qttools
+, webviewProvider ? null
+, jsProvider ? null
+, enableKioPluginSupport ? false
+, enablePlasmoidSupport  ? false
+, systemdSupport ? true }:
+
+mkDerivation rec {
+  version = "0.9.1";
+  pname = "syncthingtray";
+
+  src = fetchFromGitHub {
+    owner  = "Martchus";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "0ijwpwlwwbfh9fdfbwz6dgi6hpmaav2jm56mzxm6as50iwnb59fx";
+  };
+
+  buildInputs = [ qtbase cpp-utilities qtutilities webviewProvider jsProvider ] 
+    ++ lib.optional enableKioPluginSupport kio
+    ++ lib.optional enablePlasmoidSupport [ extra-cmake-modules plasma-framework ]
+  ;
+
+  nativeBuildInputs = [ cmake qttools ];
+
+  cmakeFlags = lib.optionals (enablePlasmoidSupport == false) ["-DNO_PLASMOID=ON"]
```

Comment:
`!enablePlasmoidSupport` (same goes for the line below)

Reply:
Done.